src/main/docs/guide/config/immutableConfig.adoc

```diff
@@ -1,4 +1,6 @@
-Since 1.3, Micronaut framework supports the definition of immutable configuration.
+Since 1.3, Micronaut framework supports the definition of immutable configuration. To enable mutable configuration, add the following dependency on Micronaut Context:
```

Comment:
```suggestion
Since 1.3, Micronaut framework supports the definition of immutable configuration. Immutable configuration with an interface requires the Micronaut Context dependency.
```